tests/datavolume_test.go

```diff
@@ -855,6 +855,12 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			switch name {
 			case "import-http":
 				dataVolume = utils.NewDataVolumeWithHTTPImportToBlockPV(dataVolumeName, "1G", url(), f.BlockSCName)
+			case "import-vddk":
+				dataVolume = createVddkDataVolume(dataVolumeName, "1Gi", vcenterURL())
+				utils.ModifyDataVolumeWithVDDKImportToBlockPV(dataVolume, f.BlockSCName)
```

Comment:
Do we want to test this with a file system import as well? The code looks like it cares what volume mode the target is so we should test file system volumes as well.

Reply:
It should already be getting tested with a filesystem import: https://github.com/kubevirt/containerized-data-importer/blob/7783203b5a027904166c172fb941fd35a72dddb7/tests/datavolume_test.go#L728

Reply:
I'm not sure I understood the original request, so I should clarify: I added cold and warm VDDK import tests to the only existing block device test section I saw in datavolume_test.go, and there were already cold and warm VDDK import tests for filesystem volumes elsewhere in the same file from a previous pull request. So now there are cold and warm VDDK tests for both block device and filesystem volumes.

Does that cover it, or were you looking for a different kind of test entirely?

Reply:
No that covers it, I just wanted to make sure we had both block and file system tests for both warm and regular vddk imports.